### PR TITLE
Nmx logger

### PR DESCRIPTION
**Pull request by Lepidopteron**

### CODE REVIEW

File: Development/ios-objc-foundation-nmxcore Tests/Tests/C-Function Helpers/NMXLoggingTest.m [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+{
+    NSString* content = [NSString stringWithContentsOfFile:[self logFilePath]
+                                                  encoding:NSUTF8StringEncoding
+                                                     error:NULL];
+    return [content hasSuffix:[string stringByAppendingString:@"\n"]];
+}
+
+- (void)testNMXLogFormat_Valid
+{
+//    XCTAssertNoThrow(NMXLog(nil));
```
Comment (thbopst) [inline Development/ios-objc-foundation-nmxcore Tests/Tests/C-Function Helpers/NMXLoggingTest.m]: Did you forget to remove the commented lines?

File: Example Swift/ios-objc-foundation-nmxcore Swift ExampleTests/ios_objc_foundation_nmxcore_Swift_ExampleTests.swift [hunk 2]
```diff
@@ -0,0 +1,36 @@
+//
```
Comment (thbopst) [inline Example Swift/ios-objc-foundation-nmxcore Swift ExampleTests/ios_objc_foundation_nmxcore_Swift_ExampleTests.swift]: Do you want to remove the generated tests?

**Review by thbopst — COMMENTED** [review]
Hi @Lepidopteron 
I added two comments. What is you opinion on that?

File: Development/ios-objc-foundation-nmxcore Tests/Tests/C-Function Helpers/NMXLoggingTest.m (hunk 1 above)
Comment (Lepidopteron) [reply Development/ios-objc-foundation-nmxcore Tests/Tests/C-Function Helpers/NMXLoggingTest.m]: I forget to put these tests back in, thank you! Already pushed them.

File: Example Swift/ios-objc-foundation-nmxcore Swift ExampleTests/ios_objc_foundation_nmxcore_Swift_ExampleTests.swift (hunk 2 above)
Comment (Lepidopteron) [reply Example Swift/ios-objc-foundation-nmxcore Swift ExampleTests/ios_objc_foundation_nmxcore_Swift_ExampleTests.swift]: 👍

**Pull request merged**